package.json

```diff
@@ -1,8 +1,22 @@
 {
-  "name": "{{ project-name }}",
-  "version": "1.0.0",
-  "description": "A template for kick starting a Cloudflare Workers project",
-  "main": "index.js",
-  "author": "{{ authors }}",
-  "license": "MIT"
+    "name": "{{ project-name }}",
```

Comment:
looks like this might need an indentation fix

Reply:
Thanks I am confused though. I ran lint on this and don't see a difference  🤷‍♀ could you put in a suggestion?

Reply:
ah, ok! it moved from 2 spaces to 4 - i'm ok with that, just didn't know if it was intentional or not. not a blocker - let's go with 4 if you're into it!